### DISCUSSION TITLE
Sonoff Zigbee Bridge MQTT all connected devices

### DISCUSSION THREAD

**Discussion by fleezer94** [arendst/Tasmota#10891] [Q&A] [answered]
hi, i have a Tasmotized sonoff zigbee bridge, and i would just like to know how can i allow mqtt messages push through the bridge to my broker for all the devices. apparently its just publishing only the coordinator and not the devices .
please a small guide will help a lot thanks :)

**Comment by sfromis** [comment, marked as answer]
Well, I did not do anything extra, after flashing the bridge and pairing the Zigbee devices. I've had cases of a device only being "partially" paired where it did not transmit MQTT payloads; then pairing again (ensuring very close distance) fixed that.
